great_expectations/data_context/types/base.py

```diff
@@ -1650,8 +1650,6 @@ def __init__(
         self._config_version = config_version
         if self.config_version is None:
             class_name = class_name or "LegacyCheckpoint"
-            if validation_operator_name is None:
-                validation_operator_name = "action_list_operator"
```

Comment:
In the line below, `self.validation_operator_name = validation_operator_name` we might want to be careful.  Specifically, set `self.validation_operator_name = validation_operator_name` only if `validation_operator_name is not None' -- and tests would have to be rerun to insure that this will work (i.e., that sometimes `validation_operator_name` will be absent from the config).

Reply:
I removed that check here because I basically moved the default behavior setting to the LegacyCheckpoint, i.e. if you instantiate a LegacyCheckpoint and then attempt to run without a validation operator set, it'll just use the "default" action_list_validation_operator (essentially a SimpleCheckpoint).